compiler/rustc_codegen_ssa/src/mir/intrinsic.rs

```diff
@@ -127,16 +128,8 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             }
 
             sym::copy_nonoverlapping => {
-                copy_intrinsic(
-                    bx,
-                    false,
-                    false,
-                    substs.type_at(0),
-                    args[1].immediate(),
-                    args[0].immediate(),
-                    args[2].immediate(),
-                );
-                return;
+                // handled explicitly in compiler/rustc_codegen_ssa/src/mir/block.rs
+                unreachable!();
             }
```

Comment:
You can completely remove this. There is already a `bug!()` for unhandled intrinsics.